pkg/kubelet/kubelet.go

```diff
@@ -911,6 +928,7 @@ func (kl *Kubelet) getPodContainerDir(podUID types.UID, ctrName string) string {
 	return newPath
 }
 
+// dirExists returns true if the path exists and represents a directory.
```

Comment:
Yeah, i was going to mark things for move, but I figured instead I would just get the godoc in now and do the moves in a follow-up.